Modix/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -150,8 +150,10 @@ public static IServiceCollection AddModix(this IServiceCollection services)
         public static IServiceCollection AddStatsD(this IServiceCollection services, IHostingEnvironment environment)
         {
             var cfg = new StatsdConfig { Prefix = "modix" };
-
-            if (!environment.IsProduction() && string.IsNullOrWhiteSpace(cfg.StatsdServerName))
+            bool.TryParse(Environment.GetEnvironmentVariable("MODIX_ENABLESTATSD"), out var enableStatsd);
```

Comment:
Consume the setting through IConfiguration with configBuilder.AddEnvironmentVariables("MODIX") if it's not already configured.

Reply:
Done